core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -102,6 +102,16 @@ sealed abstract class Aggregate extends Data {
 
   private[chisel3] def width: Width = getElements.map(_.width).foldLeft(0.W)(_ + _)
 
+  private[chisel3] def legacyAlwaysStrictConnect(that: Data)(implicit sourceInfo: SourceInfo): Unit = {
```

Comment:
`<-` is called by the firrtl spec as `partial connect`. `firrtl.<=`  is "connect". i think the semantics here are partial connect, so how about call this `firrtlPartialConnect`?

Reply:
hm but here we are issuing a firrtl.Connect so I agree that `firrtlConnect` is a good name for this function

Reply:
ok got it this is *changing* the old Chisel.:= behavior to actually emit `<=` instead of what it currently emits `<-`, and does legality check to ensure that it is OK before doing so

Reply:
I renamed this to `firrtlConnect`